src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationDetailViewController.swift

```diff
@@ -54,10 +54,3 @@ extension AppInformationDetailViewController {
 		case html = "htmlCell"
 	}
 }
-
-extension AppInformationDetailViewController: UITextViewDelegate {
-	func textView(_ textView: UITextView, shouldInteractWith url: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
-		WebPageHelper.openSafari(withUrl: url, from: self)
-		return false
-	}
-}
```

Comment:
So links within App Information will henceforth always open outside of the app context? Is this the desired effect and aligned with PO and/or design?

Reply:
I have not been able to get anything reliable out of the PO besides that the interactivity should only be in the AppInformation screens - due to this missing information, I will go with what @AdnanZiSAP suggested and make it consistent.